packages/fxa-event-broker/package.json

```diff
@@ -78,6 +79,7 @@
     "@types/chance": "^1.1.2",
     "@types/convict": "^5.2.2",
     "@types/express": "^4.17.12",
+    "@types/hapi__joi": "^15",
```

Comment:
Q: Do we need this now that we're not using `@hapi/joi`?
I see from https://github.com/sideway/joi/blob/95feacc295bc1d72412c128c76f8b1c989576c1e/package.json#L7 that the `joi` package includes its own types definitions.